File: ago-dic-2020/Luis Enrique Cázares Martínez/Practica 4 Palindromo/Palindromo.py
def palindromo(palindromo):
    palindromo = palindromo.replace(" ", "")
    if palindromo.lower() == palindromo[::-1].lower():
        return print("Si es una palabra palindrosisisima")
    else:
        return print("No es una palabra palindrosisisima")
 
cadena = input("Echate una palabra o frasse palidrosisisisma ploxs: ")
palindromo(cadena)

File: ago-dic-2020/LucelyFlores/Practica4/Palindromo.java
public class Palindromo 
{	boolean Palindromo(String palabra) 
	{	if(palabra.length() <= 1)
		{
		    return true;
		  }
		  else
		  {
			  if(palabra.charAt(0) == palabra.charAt(palabra.length()-1))
			  {
				  return Palindromo(palabra.substring(1,palabra.length()-1));
			  }
			  else
		      {
		          return false;
		      }
		  } 
	}
}